src/main/java/mate/academy/model/Book.java

```diff
@@ -0,0 +1,47 @@
+package mate.academy.model;
+
+import java.math.BigDecimal;
+
+public class Book {
+    private Long id;
+    private String title;
+    private BigDecimal price;
+
+    public Book(String title, BigDecimal price) {
+        this.title = title;
+        this.price = price;
+    }
+
+    public Book() {
+    }
+
+    public Book(Long id, String title, BigDecimal price) {
+        this.id = id;
+        this.title = title;
+        this.price = price;
+    }

```

Comment:
do you use all these constructors?

Reply:
yes, I use all 3 (so I didn't remove any of them)